zulip_botserver/zulip_botserver/input_parameters.py

```diff
@@ -40,4 +39,9 @@ def parse_args() -> argparse.Namespace:
         type=int,
         help='Port on which you want to run the Botserver. (default: %(default)d)'
     )
+    parser.add_argument(
+        '--use-env-vars', '-e',
+        action='store_true',
+        help='Optional argument to parse values through environment variables. (default: %(default)d)'
```

Comment:
Can we move this up to be under `--config-file`?  It is also now only one environment variable, not more than one, so we should name it here.

Finally, see below for a how to deal more cleanly with `--config-file`.